src/components/o-modal/index.js

```diff
@@ -48,6 +48,9 @@ class AXAModal extends BaseComponentGlobal {
   connectedCallback() {
     super.connectedCallback();
 
+    // FOUC workaround
+    this.style.display = 'block';
```

Comment:
@raphaellueckl 
@LucaMele 
That works, though it trumps styles coming from a class.
Just if you the case to control `display` by a class, better to remove the inline style, like:

```js
this.style. removeProperty('display');
```

https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/removeProperty